e2e/cypress/integration/bot_accounts/edit_bot_username_spec.js

```diff
@@ -14,20 +14,12 @@ import {getRandomId} from '../../utils';
 
 describe('Edit bot username', () => {
     let testTeam;
-    let botName;
-    let newbotName;
 
-    // # Login as admin and visit town-square
     before(() => {
         // # Set ServiceSettings to expected values
         const newSettings = {
             ServiceSettings: {
                 EnableBotAccountCreation: true,
-                DisableBotsWhenOwnerIsDeactivated: true,
-            },
-            PluginSettings: {
-                Enable: true,
-                RequirePluginSignature: false,
```

Comment:
`DisableBotsWhenOwnerIsDeactivated` and `RequirePluginSignature`  didn't work on Cloud by default.